package.json

```diff
@@ -96,13 +97,17 @@
     "@polymer/paper-listbox": "^3.0.1",
     "@polymer/paper-material": "^3.0.1",
     "@polymer/paper-menu-button": "^3.0.1",
+    "@polymer/paper-radio-button": "^3.0.1",
+    "@polymer/paper-radio-group": "^3.0.1",
     "@polymer/paper-slider": "^3.0.1",
     "@polymer/paper-spinner": "^3.0.2",
     "@polymer/paper-styles": "^3.0.1",
     "@polymer/paper-tabs": "^3.1.0",
     "@polymer/paper-toggle-button": "^3.0.1",
     "@polymer/paper-toolbar": "^3.0.1",
     "@polymer/polymer": "^3.4.1",
+    "@vaadin/vaadin-grid": "^5.6.6",
+    "@vaadin/vaadin-split-layout": "^4.3.0-alpha1",
```

Comment:
Umm... Bad news. vaadin-split-layout is stuck on very old version internally. We need to something about that in order to add this dependency.

Reply:
Ack, I'll take on the task of upgrading the internal versions to match.